src/matvis/core/beams.py

```diff
@@ -61,6 +59,12 @@ def _wrangle_beams(
     """
     # Get the number of unique beams
     nbeam = len(beam_list)
+    beam_list = [
+        beam if isinstance(beam, BeamInterface) else BeamInterface(beam)
+        for beam in beam_list
+    ]
```

Comment:
BeamInterface can now be constructed from a BeamInterface object, so you don't need this conditional any more.

Reply:
Good catch!